### PR TITLE
fix: Save and Dispose Rx subscriptions in FeedFragment

### LINKED ISSUE #2120: Save and dispose RxSubscription for Observables in FeedFragment
**Actual Behaviour**

Currently the Subscriptions are not saved when creating and subscribing for observables in FeedFragment.

**Expected Behaviour**

They should be saved in a variable and disposed off in ```onStop```

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by dr0pdb**
Fixes #2120 
Changes: Saved subscriptions in Disposables and disposed them in FeedFragment.

Screenshots for the change: NA

@iamareebjamal Please review.

### CODE REVIEW

File: android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java [hunk 1]
```diff
@@ -128,7 +129,10 @@ public void networkAvailable() {
                 // Network is available
                 swipeRefreshLayout.setRefreshing(true);
                 if (SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_ID, null) == null)
-                    APIClient.getFacebookGraphAPI().getPageId(SharedPreferencesUtil.getString(ConstantStrings.FACEBOOK_PAGE_NAME, null),
+                    if (refreshFeedDisposable != null && !refreshFeedDisposable.isDisposed()) {
```
[the remark this answers is not in the document]
Comment (srv-twry) [reply android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java]: @iamareebjamal Should i combine them ?

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java]: Yes, please combine these

File: android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java [hunk 2]
```diff
@@ -40,7 +40,8 @@
     private FeedAdapter feedAdapter;
     private List<FeedItem> feedItems;
     private ProgressDialog downloadProgressDialog;
-    private Disposable feedLoaderDisposable;
+    private Disposable feedDownloaderDisposable;
+    private Disposable refreshFeedDisposable;
```
Comment (iamareebjamal) [inline android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java]: Use a final CompositeDisposable instead. This makes us remove the null checks as well

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by dr0pdb** [comment]
Changes: Using final CompositeDisposable, Combined two ```if``` statements into one.

@iamareebjamal Please review.

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**